server/build-plugins.js

```diff
@@ -42,15 +85,30 @@ const prepareModules = async (plugins = [], local = true) => {
   let exportsText = 'export default {';
   let installPackages = [];
 
+  // Create the index.js files for exposing the plugins
   plugins.forEach(name => {
     const packageName = getPackageName(name);
-    const camelized = camelize(packageName);
-    const modulePath = local ? `./${packageName}` : packageName;
-    if (!local) installPackages.push(name);
-    importsText += `import * as ${camelized} from '${modulePath}';\n`;
-    exportsText += `${camelized},`;
+    try {
+      const camelized = camelize(packageName);
+      let modulePath;
+      if (fs.existsSync(resolve(pluginsPath, 'local', packageName)) && local)
```

Comment:
This is a lot of operations for a single line without a comment

Reply:
Adding comments.